### PR TITLE
Creates a documentation for the can variables

### PR DESCRIPTION
**Pull request by jeffhsta**
Can variables:
- ENGINE_HOURS
- ENGINE_SPEED
- DRIVING_DIRECTION

### CODE REVIEW

**Comment by waldemarnt** [comment]
👍

File: source/includes/_can-variables.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Can variables
+
+## Description
+
+Name | Description | Standard Unit
+---- | ----------- | -------------
+ENGINE_HOURS | Total hours of equipment engine | Seconds
+ENGINE_SPEED | Engine speed of equipment | Rpm
```
Comment (glauco) [inline source/includes/_can-variables.md]: s/Rpm/RPM/

Comment (glauco) [inline source/includes/_can-variables.md]: 👍

File: source/includes/_can-variables.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Can variables
+
+## Description
+
+Name | Description | Standard Unit
+---- | ----------- | -------------
+ENGINE_HOURS | Total hours of equipment engine | Seconds
+ENGINE_SPEED | Engine speed of equipment | RPM
```
Comment (glauco) [inline source/includes/_can-variables.md]: 👍

**Comment by glauco** [comment]
LGTM!

**Comment by bltavares** [comment]
LGTM

**Pull request merged**